### ISSUE TITLE
As a NAVSA admin, I want users to authenticate via navsa.org

### ISSUE THREAD

**Issue by jamiefolsom** [performant-software/Annotation-Studio#73]
Updated: 
- When the switch to SSO is ready to be completed, the "Not a NAVSA member?" option should be removed and the "Log in" nav bar item should directly initiate authentication through NAVSA
- Language in navsa.org's login view needs to communicate to different types of users what NAVSA is and why they're being asked to sign in through it.
- Once the SSO feature is complete, users should have to log into COVE Studio and COVE Editions via navsa.org, to confirm that they do have a membership.
- Even users who already have accounts in those other systems should be prompted to authenticate via navsa once.
- This is a very low priority, until all other work is done, on SSO and on the tools themselves.

**Comment by jamiefolsom** [comment]
Completed
